test-app/src/main/ml-modules/root/javascript/callOptic.sjs

```diff
@@ -0,0 +1,15 @@
+const op = require('/MarkLogic/optic');
+
+/*
```

Comment:
This seems like something that would be more likely to be read in the CONTRIB guide, which would have a section on how to test stepping into built-in MarkLogic code. And it wouldn't need to be duplicated in the callOptic.xqy file either.